charts/nextcloud/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: nextcloud
-version: 2.11.3
-appVersion: 22.2.3
+version: 2.12.0
+appVersion: 23.0.0
```

Comment:
can you update to 23.0.1?

Reply:
@mrueg I'm not seeing 23.0.1 tag in dockerhub right now.  Am I just missing something?

Reply:
You are correct, I was hoping the nextclouders had pushed it already. :)

Reply:
now I updated to 23.0.2

Reply:
@marcelloceschia can you resolve the merge conflicts?